ledger/ledger.go

```diff
@@ -3,11 +3,14 @@ package ledger
 import (
 	"bytes"
 	"encoding/hex"
+	"encoding/json"
 	"fmt"
 
 	"github.com/onflow/flow-go/module"
 )
 
+const CacheSize = 1000
```

Comment:
This shouldn't be here in my view, should be part of the complete ledger.

Reply:
Yeah, you're right